05/Weather.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {getWeatherData} from "./API.js";
+
+
+class Weather extends React.Component {
+    state = {
+        data: null,
+        key: '3be1579ab3be42d78e26bfae3f3b9db8',
```

Comment:
Docelowo `key` warto trzymać w osobnym pliku i dodać go do `.gitignore` :) (wiem, że teraz to tylko ćwiczenia, ale tak na pszyszłość)